en/physics-2d/physics-2d-rigid-body.md

```diff
@@ -1,126 +1,173 @@
+Moving velocity damping factor, the larger the value the slower the object moves, can be used to simulate the effect of air friction and so on.
 
 ```ts
-// Get the moving speed attenuation factor
+// get linear damping
 const damping = rigidbody.linearDamping;
-// Set the moving speed attenuation factor
+// set linear damping
 rigidbody.linearDamping = damping;
 ```
 
-To get the velocity of a point on a RigidBody, such as a box that rotates forward and touches the wall. It may be desirable to get the velocity of the box at the point of the collision. You can get it by  `getLinearVelocityFromWorldPoint`.
+If you want to get the velocity of a point on a rigid body, you can get it with `getLinearVelocityFromWorldPoint`. For example, if a box is spinning and flying forward and hits a wall, you may want to get the velocity of the box at the point where the collision occurred.
 
 ```ts
 const velocity = rigidbody.getLinearVelocityFromWorldPoint(worldPoint);
 ```
 
-Or, pass in a `Vec2` object as the second argument to get the return value in order to use the cached object to store this value. This avoids creating too many objects to improve performance.
+Or pass a `Vec2` object as the second argument to receive the return value, so you can use your cached object to receive the value and avoid creating too many objects for efficiency.
 
-> __Note__: the `get` method of RigidBody provides an out parameter to receive the function return value.
+**The rigid get methods all provide out arguments to receive the function return value.**
 
 ```ts
-const velocity = v2();
+const velocity = new Vec2();
 rigidbody.getLinearVelocityFromWorldPoint(worldPoint, velocity);
 ```
 
 ### Angular Velocity
 
 ```ts
-// Get the angular velocity
+// get angular velocity
 const velocity = rigidbody.angularVelocity;
-// Set the angular velocity
+// set angular velocity
 rigidbody.angularVelocity = velocity;
 ```
 
-Travel speed decay coefficient, the larger the value the slower the object moves, and can be used to simulate effects such as air friction.
+Angular velocity damping factor, same as the linear velocity damping factor.
 
 ```ts
-// Get the angular damping
+// get angular damping factor
 const damping = rigidbody.angularDamping;
-// Set the angular damping
+// set angular damping factor
 rigidbody.angularDamping = damping;
 ```
 
-### Rotation, position and scaling
+### Rotation, Translation and Scaling
 
-Rotation, position and scaling are the most commonly used transform in game development, and almost every node has these properties set. In the physics system, the system will automatically synchronize these properties of the node to __Box2D__ corresponding properties.
+Rotation, translation and scaling are the most common features used in game development, and almost every node has these properties set. In the physics system, the system automatically synchronizes these properties of the node with the corresponding properties in Box2D.
```

Comment:
"these properties set" does not seem right.
"In the physics system, the system" seems tedious.